backend/PythonClient/multirotor/monitor/point_deviation_monitor.py

```diff
@@ -22,6 +26,8 @@ def __init__(self, mission, deviation_percentage=15):
         self.deviation_percentage = deviation_percentage
         self.dt = 0.02  # 50 Hz
         self.point_queue = None
+        self.air_sim_app = AirSimApplication() 
+        self.grapher = ThreeDimensionalGrapher()
```

Comment:
Can you explain the purpose of this? `ThreeDimensionalGrapher` is already initialized.